src/lib/constants.js

```diff
@@ -103,3 +103,9 @@ export const PREVIEW_SCRIPT_NAME = 'preview.js';
 
 export const FILE_OPTION_FILE_VERSION_ID = 'fileVersionId';
 export const FILE_OPTION_START = 'startAt';
+
+// Query parameter for requesting encoded PDF representations
+export const QUERY_PARAM_PDF_ENCODING = 'encoding';
+export const PDFEncodingTypes = {
```

Comment:
Should this use the same `STYLE_OF_NAMING` as the other constants?

Reply:
Good catch